sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/TranslationOperationLiveTests.cs

```diff
@@ -0,0 +1,214 @@
+            var input = new DocumentTranslationInput(source, targets);
+            var operation = await client.StartTranslationAsync(input);
+
+            await operation.WaitForCompletionAsync();
+
+            Assert.IsTrue(operation.HasCompleted);
+            Assert.IsTrue(operation.HasValue);
+            Assert.AreEqual(1, operation.DocumentsTotal);
+            Assert.AreEqual(1, operation.DocumentsSucceeded);
+            Assert.AreEqual(0, operation.DocumentsFailed);
+            Assert.AreEqual(0, operation.DocumentsCancelled);
+            Assert.AreEqual(0, operation.DocumentsInProgress);
+            Assert.AreEqual(0, operation.DocumentsNotStarted);
+        }
+
+        [RecordedTest]
+        public async Task SingleSourceSingleTargetWithSuffixTest()
+        {
+            Uri sourceUri = await CreateSourceContainerAsync(twoDocumentsList);
+            Uri targetUri = await CreateTargetContainerAsync();
+
+            var client = GetClient();
+
+            var filter = new DocumentFilter
+            {
+                Suffix = "1.txt"
+            };
+            var source = new TranslationSource(sourceUri)
+            {
+                Filter = filter
+            };
+            var targets = new List<TranslationTarget> { new TranslationTarget(targetUri, "fr") };
+            var input = new DocumentTranslationInput(source, targets);
+            var operation = await client.StartTranslationAsync(input);
+
+            await operation.WaitForCompletionAsync();
+
+            Assert.IsTrue(operation.HasCompleted);
+            Assert.IsTrue(operation.HasValue);
+            Assert.AreEqual(1, operation.DocumentsTotal);
+            Assert.AreEqual(1, operation.DocumentsSucceeded);
+            Assert.AreEqual(0, operation.DocumentsFailed);
+            Assert.AreEqual(0, operation.DocumentsCancelled);
+            Assert.AreEqual(0, operation.DocumentsInProgress);
+            Assert.AreEqual(0, operation.DocumentsNotStarted);
+        }
+
+        [RecordedTest]
+        public async Task SingleSourceSingleTargetListDocumentsTest()
+        {
+            Uri sourceUri = await CreateSourceContainerAsync(oneDocumentList);
+            Uri targetUri = await CreateTargetContainerAsync();
+
+            var client = GetClient();
+
+            var filter = new DocumentFilter
+            {
+                Suffix = "1.txt"
+            };
+            var source = new TranslationSource(sourceUri)
+            {
+                Filter = filter
+            };
+            var targets = new List<TranslationTarget> { new TranslationTarget(targetUri, "fr") };
+            var input = new DocumentTranslationInput(source, targets);
+            var operation = await client.StartTranslationAsync(input);
+
+            await operation.WaitForCompletionAsync();
+            var documents = operation.GetAllDocumentStatusesAsync();
+
+            int documentsCount = 0;
```

Comment:
what is the purpose of this variable?

Reply:
@maririos To check that the number of documents returned is correct. Is there a way to get the number of elements in an AsyncPageable?

Reply:
yes u can use https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/TestAsyncEnumerableExtensions.cs